Sources/Purchasing/StoreKit2/StoreKit2TransactionFetcher.swift

```diff
@@ -279,22 +288,42 @@ extension StoreKit2TransactionFetcher {
         }
     }
 
-    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
-    private var appTransaction: SK2AppTransaction? {
+    @available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *)
+    private var refreshedAppTransaction: SK2AppTransaction? {
```

Comment:
The unified diff of this is a bit weird.

`appTransaction` was modified into the `appTransaction(refreshPolicy:)` method below.

and `refreshedAppTransaction` was added as a helper to fetch an `AppTransaction` via `StoreKit.AppTransaction.refresh()`